Succulent/Classes/Succulent.swift

```diff
@@ -48,27 +64,19 @@ public class Succulent : NSObject, URLSessionTaskDelegate {
     private var currentTrace = NSMutableOrderedSet()
     private var recordedKeys = Set<String>()
     private var ignoreExpressions: [NSRegularExpression] = []
-    
-    public override init() {
-        super.init()
-        
-        createDefaultRouter()
-    }
 
-    public convenience init(traceUrl: URL, baseUrl: URL? = nil, ignoreVersioningRequests: [String] = []) {
-        self.init()
+    public convenience init(replayFrom traceUrl: URL? = nil, passThroughBaseUrl baseUrl: URL? = nil, configuration: Configuration? = nil) {
```

Comment:
Could we please remove the default from `replayFrom`? I'd like it to be clear which mode Succulent is in.